lib/manager/pipenv/extract.js

```diff
@@ -47,14 +48,8 @@ function extractFromSection(pipfile, section, registryUrls) {
 
   if (section === 'packages') {
     Object.keys(oneSection).forEach(key => {
-      if (typeof oneSection[key] === 'string') return;
-
-      const isObject =
-        !Array.isArray(oneSection[key]) &&
-        typeof oneSection[key] === 'object' &&
-        oneSection[key] !== null;
-
-      if (!isObject) oneSection[key] = {};
+      if (is.string(oneSection[key])) return;
+      if (!is.object(oneSection[key])) oneSection[key] = {};
```

Comment:
Do you have any examples of pipfiles where the value is not a string but also not an object?

Reply:
No, I don't. Please give me one and I will update tests.